web_interface.js

```diff
@@ -1066,6 +1067,101 @@ async function main() {
 		}
 	})
 
+	app.get(endpoint + "/Logs", function (req, res) {
+		var authHeader = req.get("Authorization")
+		var authToken
+		if (authHeader) {
+			if (authHeader.indexOf(" ") == -1) {
+				authToken = authHeader
+			} else {
+				authHeader = authHeader.split(" ")
+				authToken = authHeader[1]
+			}
+		} else {
+			res.set("Authorization", "Basic base64_auth_token")
+			return res
+				.type("txt")
+				.status(401)
+				.send(
+					'Access denied: invalid Authorization Header format. Correct format: "Authorization: Basic base64_auth_token"'
+				)
+		}
```

Comment:
I absolutely sure that this part can be carried out from each route, avoiding terrible code repetition!!!

Reply:
Yes that is true.  We will add that to the list of things to address.